README.md

```diff
@@ -141,4 +160,4 @@ Please read our [contributing guidelines](/.github/CONTRIBUTING.md) before submi
 
 ## License
 
-[MIT](/LICENSE)
```

Comment:
What's happening here? Unintended whitespace change? intended whitespace correction?

Reply:
That was my guess. Didn't seem to hurt anything so I left it. Could be a windows vs linux line ending thing.

Reply:
Cool, I'm sure it can't hurt, just weird :)